frontend/src/elements/components/UserData.js

```diff
@@ -58,18 +84,21 @@ class UserData extends React.PureComponent {
         <div>Controller: {status.controller}</div>
         <br />
         <b>Processors</b>
-        <ul>
-          {status.processors.map(p => (
-            <div key={p.id} className={styles.processor}>
-              <span
-                className={[styles.status, p.status ? styles.isActive : styles.isInactive].join(
-                  ' '
-                )}
-              />
-              <span>{p.id}</span>
+        {processorsList.map(p => (
+          <div key={p.id} className={styles.processor}>
+            <div>
+              <img src={p.logoUrl} alt={p.name} />
             </div>
-          ))}
-        </ul>
+            <div>{p.name}</div>
+            <div>{p.description}</div>
+            <div>
+              <b>Status</b>: {p.status}
+            </div>
+            <div>
```

Comment:
the scopes are not currently used/implemented, lets not display them for now

Reply:
👍